DatabaseAndReporting/Integration/02_ManageQueryCategories.md

```diff
@@ -0,0 +1,218 @@
+[!!Manage the queries](./01_ManageQueries.md)
+[!!Manage the user rights](./05_ManageUserRights.md)
+
+# Manage the query categories
+
+Query categories serve the logical organization of the managed queries. The managed queries are then organized in a tree structure of folders and sub-folders.
+
+Query categories can be created, edited, deleted, and divided into sub-categories. All individual managed queries can, in turn, be assigned to a category and/or a sub-category, see [Create a query](./01_ManageQueries.md#create-a-query). Query category read and save rights can also be restricted or granted based on the user group, see [Grant access rights to a user group](./05_ManageUserRights.md#grant-access-rights-to-a-user-group).
+
+
+## Create a query category
+
+Create a query category for new managed queries you want to assign to a certain category.
+
+#### Prerequisites 
+
+No prerequisites to fulfill.
+
+#### Procedure
+
+*Database and reporting > Settings > Tab QUERY CATEGORIES*
+
+![Query categories](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/QueryCategories.png "[Query categories]")
+
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    A plus sign is displayed in the query category column.
+
+    ![Add query category](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/AddQueryCategory.png "[Add query category]")
+
+2. Click the plus sign in the query category column.    
+    The *Add element* window is displayed in the query category column.
+
+    ![Add element](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/AddElementCategory.png "[Add element]")
+
+3. Enter a name for the query category in the *Name* field.
+
+4. Enter a key for the query category in the *Key* field.  
+
+[comment]: <> (Key wird hier irgendwann abgebaut)
+
+5. Click the [SAVE] button.   
+    The query category has been created. The *Add element* window is closed. The new query category is displayed in the query category column.
+
+    ![Query category created](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/QueryCategoryCreated.png "[Query category created]")
+
+    > [Info] To add an additional query category, repeat the steps 1 to 5. To add a query sub-category, see [Create a query sub-category](#create-a-query-sub-category).
+
+6. Click the [SAVE] button in the upper right corner.   
+    The *Submitting data...* view is displayed shortly while saving. 
+
+    ![Submitting data](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/SubmittingData.png "[Submitting data]")
+
+    The new query category has been saved to the query categories tree, which is displayed in the *Queries* view.
+    > [Info] It may be necessary to press the **F5** key to initialize the *Core1 Platform* for the changes to be displayed in the *Queries* view. 
```

Comment:
```suggestion
    > [Info] It may be necessary to press the **F5** key to initialize the *Core1 Platform* to display the changes in the *Queries* view. 
```